app/workers/metrics_collection_worker.rb

```diff
@@ -2,8 +2,8 @@ class MetricsCollectionWorker
   include Sidekiq::Worker
 
   def perform
-    Metrics::ContentChangeExporter.call
-    Metrics::DigestRunExporter.call
-    Metrics::MessageExporter.call
+    MetricsCollectionWorker::ContentChangeExporter.call
+    MetricsCollectionWorker::DigestRunExporter.call
+    MetricsCollectionWorker::MessageExporter.call
```

Comment:
Oh and I just realised that you don't actually need to explicitly navigate to MetricsCollectionWorker, just calling the classes straight up should be enough, e.g.

`MetricsCollectionWorker::ContentChangeExporter.call` -> `ContentChangeExporter.call`

Reply:
Indeed. At the time I thought it might be helpful to be explicit, but it's also nice to be concise, and [somewhat consistent](https://github.com/alphagov/email-alert-api/blob/master/app/services/delivery_request_service.rb#L15). Updated.